core/src/test/scala/integration/kafka/api/PlaintextConsumerSubscriptionTest.scala

```diff
@@ -223,6 +225,27 @@ class PlaintextConsumerSubscriptionTest extends AbstractConsumerTest {
     assertEquals(0, consumer.assignment.size())
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testSubscribeInvalidTopic(quorum: String, groupProtocol: String): Unit = {
+    // Invalid topic name due to space
+    val invalidTopicName = "topic abc"
+    val consumer = createConsumer()
+
+    // subscribe invalid topic
```

Comment:
nit: I would remove, the line below seems clear enough

Reply:
You mean to delete line 234, right?